### DISCUSSION TITLE
Twitch

### DISCUSSION THREAD

**Discussion by johan-perso** [ReVanced/revanced-discussions#606] [Question] [answered]
It can be cool to have a patch to remove ads on twitch stream

**Comment by blankLines** [comment]
I would like to join this one. there is no proper developed app for android out there that removes ads.

https://github.com/bttv-android/bttv/issues/87#issuecomment-855405919

**Comment by timschneeb** [comment, marked as answer]
I've submitted an ad-blocking patch for Twitch: https://github.com/revanced/revanced-patches/releases/tag/v2.109.0
It should be visible in the manager soon.

**Reply by ghost** [reply]
Is there a specific version requirement, because I cannot get it to block ads at all
